api/cms.go

```diff
@@ -149,3 +164,34 @@ func concludeRequestFile(c *gin.Context, fileName string, err *HttpError) {
 		err.Abort(c)
 	}
 }
+
+func handleItemsRequest(filePattern string, language string) ([]string, *HttpError) {
+	for _, lang := range consts.I18N_LANG_ORDER[language] {
+		pattern := fmt.Sprintf(filePattern, lang)
+		if files, err := filepath.Glob(pattern); err == nil {
+			return files, nil
+		}
+	}
+
+	return nil, NewNotFoundError()
+}
+
+// responds with File(s) content or aborts the request with the given error.
+func concludeRequestFiles(c *gin.Context, fileNames []string, err *HttpError) {
+	if err != nil || len(fileNames) == 0 {
+		err.Abort(c)
+	}
+
+	content := make([]string, len(fileNames))
```

Comment:
Correct me if I'm wrong but initializing a slice with non zero capacity will populate the slice up to that capacity with empty values. Later on `append` will start at index capacity+1.
So you either set directly (with index) to each cell or use append. (line 194)

Reply:
done